src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js

```diff
@@ -1,94 +1,90 @@
-            // ignore these searches for search auditing
+      var formatQuery = function(query) {
+         var _query = _.isArray(query) ? query : [query];
+         if (_.isArray(_query)) {
+            var count = 0;
+            _.each(_query, function($q) {
+               $q = elasticSearchFields.convertQuery($q);
+               _query[count] = $q;
+               ++count;
+            });
+         } else {
+            _query = elasticSearchFields.convertQuery(_query);
+         }
+         return _query;
+      };
 
-            var timeRange = timefilter.getBounds();
-            var timeFrom = timeRange.min;
-            var timeTo = timeRange.max;
+      var logQuery = function(query) {
+      // ignore these searches for search auditing
+         var timeRange = timefilter.getBounds();
+         var timeFrom = timeRange.min;
+         var timeTo = timeRange.max;
 
-            if (!timeFrom || !timeTo) {
-                return;
-            }
+         if (!timeFrom || !timeTo) {
+            return;
+         }
 
-            timeFrom = moment(timeFrom).format(FORMAT_STRING);
-            timeTo = moment(timeTo).format(FORMAT_STRING);
+         timeFrom = moment(timeFrom).format(FORMAT_STRING);
+         timeTo = moment(timeTo).format(FORMAT_STRING);
 
-            if (query === previousQuery.query &&
-                timeFrom === previousQuery.fromDate &&
-                timeTo === previousQuery.toDate) {
-                return;
-            }
-            previousQuery = {
-                query    : query,
-                fromDate : timeFrom,
-                toDate   : timeTo
-            };
-            Restangular
-                .all('audit/')
-                .post({
-                    query    : query,
-                    from     : timeFrom,
-                    to       : timeTo
-                })
-                .then(function(response){
-                    if (response.error){
-                        var error = response.message || 'An unknown error occured';
-                        console.log('Search Audit Error:', error);
-                    }
-                });
-        };
+         if (query === previousQuery.query &&
+            timeFrom === previousQuery.fromDate &&
+            timeTo === previousQuery.toDate) {
+            return;
+         }
+         previousQuery = {
+            query    : query,
+            fromDate : timeFrom,
+            toDate   : timeTo
+         };
 
-        return {
-            logAndCapitalize : function(query) {
-                return elasticSearchFields.fetchMapping().then( function(hasFieldMap) {
-                    var capitalizedQuery = formatQuery(query.query_string.query)[0];
-                    logQuery(capitalizedQuery);
-                    // seems like we need to create a new object to force angular to reflect the
-                    // query change in the view
+         $http.put('/api/audit/search', 
```

Comment:
the API route can be a const at the top with the others. 

weird formatting makes it hard to read... Maybe try something like this?
```JavaScript        
         $http.put(SEARCH_URI,  { 
            query: query,
            from: timeFrom,
            to: timeTo
         })
         .then(
             function (data) { }, 
             function (error) {
                var error = response.message || 'An unknown error occured';
                console.log('Search Audit Error:', error);
             }
         );
```

Reply:
Changed the route to a const, and beautified the formatting.